### PR TITLE
ePJS User set up

### PR DESCRIPTION
**Pull request by katbow**
Sets up multi db for epjs data #453 

Adds schema and migration for SLaM SUs #455 

Adds all the seed data for adding epjs users :tada #458

### CODE REVIEW

File: web/models/epjs_user.ex [hunk 1]
```diff
@@ -0,0 +1,23 @@
+defmodule Healthlocker.EPJSUser do
+  use Healthlocker.Web, :model
+
+  schema "epjs_users" do
+    field :patient_id, :integer
```
Comment (tonydaly) [inline web/models/epjs_user.ex]: Just thinking that in the ePJS DB this is actually `Patient_ID`, so we might have to use a string here and keep the casing consistent?

Comment (katbow) [inline web/models/epjs_user.ex]: alright cool, I'll change it to the same capitalisation. What do you mean use a string here? Can't we still do an atom like `:Patient_ID`? or do you mean the type?

Comment (tonydaly) [inline web/models/epjs_user.ex]: Sorry, that was confusing, yeah we can just use an atom, it _should_ be fine.

**Comment by katbow** [comment]
@tonydaly one question: I have the `prod.exs` db the same right now. Should we change this & set up a second one on heroku?

**Comment by tonydaly** [comment]
Hmmm good question...we should be able to use the same DB instance, but create another `database` with a different name. This should be the last thing in the `DATABASE_URL`.

**Comment by katbow** [comment]
okay so the prod database is set up now & capitalisation is fixed. I added the required env variables on heroku 🎉

**Pull request merged**